### ISSUE TITLE
support "Feature for Feature Variation" custom parameter

### ISSUE THREAD

**Issue by anthrotype** [googlefonts/glyphsLib#614]
Glyphs.app has a "Feature for Feature Variation" to allow changing the feature tag used for GSUB FeatureVariations (bracket trick) from the default "rvrn" to a user-defined feature tag.

one problem with implementing support for this in glyphsLib is that

> designspace only has processing=[first|last], and uses rvrn and rclt respectively.
Unlike that Glyphs.app custom parameter, it doesn't let you specify an arbitrary feature tag.

So if user sets that `Feature For Feature Variation` to a tag that isn't either rvrn nor rclt, besides setting rules processing="last" inthe designspace file, we will have to also store that custom feature tag in the designspace.lib as a private key, and see how we can wire it up to ufo2ft and fontTools.varLib somehow.

Or maybe we just say we only support "rclt" as alternative non-rvrn feature tag for now, as that fits the designspace model and current library support.
And refine later.

**Comment by anthrotype** [comment]
Forgot to mention, this custom parameter is important because it allows to use a feature (such as "rclt") that is _not_ "rvrn", thus allows to trigger the processing of FeatureVariations substitution rules as the _last_ lookup (whereas rvrn is always processed _first_ as per OT spec):

https://github.com/googlefonts/fontmake/issues/641

**Comment by schriftgestalt** [comment]
I changed the default to `rlig` for the next version of Glyphs.

**Comment by anthrotype** [comment]
why not "rclt" like fonttools uses?

**Comment by schriftgestalt** [comment]
I can change that if it is preferred.

**Comment by anthrotype** [comment]
yes, it would be good if we used the same feature tag. @justvanrossum opted for "rclt" in fontTools.

**Comment by justvanrossum** [comment]
See also: https://github.com/fonttools/fonttools/pull/2080
